docs/reference/contrib/modeladmin/create_edit_delete_views.rst

```diff
@@ -150,3 +150,12 @@ See the following part of the docs to find out more:
 
 See the following part of the docs to find out more:
 :ref:`modeladmin_overriding_views`
+
+.. _modeladmin_exclude_fields:
+
+-----------------------------------
+``ModelAdmin.exclude_fields``
+-----------------------------------
+
+**Expected value**: An array of Model fields to be excluded from Create/Edit pages.
```

Comment:
The examples in the [IndexView docs](https://github.com/springload/wagtail/blob/491b643ed95d8ec80064c018133cc993e78e0bac/docs/reference/contrib/modeladmin/indexview.rst) are probably a better guide for one. I would stop at explaining what the value is in the first line e.g.

`A list or tuple of field names for your model`

 then the have something like this for the description:

`When using CreateView or EditView to create or update model instances, this value will be passed to the edit form, so that any named fields will be excluded from the form. This is particularly handy if you're registering ModelAdmin classes for models from third-party apps, where defining panel configurations on the Models is less straight forward.`